jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/screens/chunking/Chunk.kt

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (C) 2020-2022 Wycliffe Associates
+ *
+ * This file is part of Orature.
+ *
+ * Orature is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Orature is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Orature.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.wycliffeassociates.otter.jvm.workbookapp.ui.screens.chunking
+
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import com.jfoenix.controls.JFXButton
+import com.sun.javafx.util.Utils
+import java.text.MessageFormat
+import javafx.animation.AnimationTimer
+import javafx.geometry.Pos
+import org.kordamp.ikonli.javafx.FontIcon
+import org.wycliffeassociates.otter.jvm.controls.styles.tryImportStylesheet
+import org.wycliffeassociates.otter.jvm.controls.waveform.MarkerPlacementWaveform
+import org.wycliffeassociates.otter.jvm.controls.waveform.MarkerTrackControl
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import org.wycliffeassociates.otter.jvm.workbookapp.ui.viewmodel.ChunkingViewModel
+import org.wycliffeassociates.otter.jvm.workbookapp.ui.viewmodel.ChunkingWizardPage
+import tornadofx.Fragment
+import tornadofx.action
+import tornadofx.borderpane
+import tornadofx.button
+import tornadofx.get
+import tornadofx.hbox
+
+class Chunk : Fragment() {
+
+    val playIcon = FontIcon("mdi-play")
+    val pauseIcon = FontIcon("mdi-pause")
+    private val nextIcon = FontIcon("gmi-skip-next")
+    private val previousIcon = FontIcon("gmi-skip-previous")
```

Comment:
Can we use ` FontIcon(MaterialDesign.MDI_PLAY/...) ` enums for all these?
https://kordamp.org/ikonli/cheat-sheet-materialdesign.html